run_agent.sh

```diff
@@ -7,9 +7,6 @@ cleanup() {
     echo "Tendermint terminated"
 }
 
-# Load env vars
-source .env
```

Comment:
we are not loading .env variables anymore ?

Reply:
Yes, but if you scroll down you'll see that this line was duplicated